VictoriaAlbert/APIRequestManager.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  APIRequestManager.swift
+//  VictoriaAlbert
+//
+//  Created by John Gabriel Breshears on 11/10/16.
+//  Copyright © 2016 John Gabriel Breshears. All rights reserved.
+//
+
+import Foundation
+
+class APIRequestManager {
+    
+    static let manager = APIRequestManager()
+    private init() {}
+    
+    func getData(searchKey: String, callback: @escaping (Data?) -> Void) {
+      
+        // Mabye instead of banging this, unwrap safely
+        let myURL = URL(string: "http://www.vam.ac.uk/api/json/museumobject/search?q=\(searchKey)")!
+        
+        
+        
+        let session = URLSession(configuration: URLSessionConfiguration.default)
+        session.dataTask(with: myURL) { (data: Data?, response: URLResponse?, error: Error?) in
+            if error != nil {
+                print("Error durring session: \(error)")
+            }
+            guard let validData = data else { return }
+            callback(validData)
+            }.resume()
+    }
+    //Okay for this to work I need to somehow get only the first 6 digits of the primary id or small id
+    func getLittlePicture(primaryImageId: String, smallId: String, callback: @escaping (Data?) -> Void) {
```

Comment:
This is good code factoring and thinking ahead about loading the image, even though you didn't get the chance to call the method. You could go one step further and call getData within it instead of re-calling session.dataTask.